### DISCUSSION TITLE
Storage and Caching strategies

### DISCUSSION THREAD

**Discussion by adnankhan-neo** [NeoSOFT-Technologies/rest-node-nestjs#52] [Ideas]
Identifying the correct storage options for assets and media files like s3 or cloud storage is very important aspect of any project architecture. In addition to this deciding on caching strategies also plays key role in project performance in long run

**Comment by MohsinNeosoft** [comment]
Hi, we can manage the content cache policy for GCP by adding headers in particular Load Balancers. By default it is 1 day and we can increase it up to 1 year

**Comment by santoshshinde2012** [comment]
Comparing Redis and Memcached

https://aws.amazon.com/elasticache/redis-vs-memcached/

REDIS VS MEMCACHED: WHICH ONE TO CHOOSE?

https://www.imaginarycloud.com/blog/redis-vs-memcached/

**Reply by santoshshinde2012** [reply]
![image](https://user-images.githubusercontent.com/6427881/131854858-48dcc7c9-38c0-43b2-8d81-03a99d316058.png)
